console/backend/src/test/java/org/frankframework/management/web/ConfigurationStatusTest.java

```diff
@@ -0,0 +1,173 @@
+package org.frankframework.management.web;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.doCallRealMethod;
+
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+import org.frankframework.management.Action;
+import org.springframework.http.MediaType;
+import org.springframework.messaging.Message;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+@ContextConfiguration(classes = {WebTestConfiguration.class, ConfigurationStatus.class})
+public class ConfigurationStatusTest extends FrankApiTestBase {
+
+	@Test
+	public void getAdapters() throws Exception {
+		testActionAndTopicHeaders("/adapters", "ADAPTER", "GET");
+	}
+
+	@Test
+	public void getAdapter() throws Exception {
+		testActionAndTopicHeaders("/configurations/configuration/adapters/name", "ADAPTER", "FIND");
+	}
+
+	@Test
+	public void getAdapterHealth() throws Exception {
+		testActionAndTopicHeaders("/configurations/configuration/adapters/name/health", "HEALTH", null);
+	}
+
+	@Test
+	public void updateAdapters() throws Exception {
+		ArgumentCaptor<Message<Object>> requestCapture = ArgumentCaptor.forClass(Message.class);
+		doCallRealMethod().when(outputGateway).sendAsyncMessage(requestCapture.capture());
+
+		mockMvc.perform(MockMvcRequestBuilders.put("/adapters")
+						.content("{ \"action\": \"start\", \"adapters\": [\"adapter1\", \"adapter2\"] }")
+						.accept(MediaType.APPLICATION_JSON)
+						.contentType(MediaType.APPLICATION_JSON))
```

Comment:
Kan je deze test herhalen met in de json voor de adapternamen de configuratie naam? (zoals die in de backend code ook uitgelezen wordt? Dus config/adapter)